support/fixtures/factories/file.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const factory = require('rosie').Factory;
+const faker = require('faker');
+
+factory
+  .define('file')
+  .option('fromServer', false)
+  .attrs({
+    createdAt: () => faker.date.past().toISOString(),
+    contentType: () => faker.system.mimeType(),
+    description: () => faker.hacker.phrase(),
+    filename: () => `${faker.hacker.noun()}.${faker.system.commonFileExt()}`,
```

Comment:
What about `() => faker.system.commonFileName()`?

Reply:
Sure, wasn't aware of that one!